tests/conditional_compilation/test_collect.py

```diff
@@ -31,13 +54,12 @@ def test_cc_collect(test_id, model, sea_runtool, benchmark_app, collector_dir, a
             str(sea_runtool),
             f"--output={out}",
             f"--bindir={collector_dir}",
-            "--app_status",
             "!",
-            str(benchmark_app),
-            "-d=CPU",
+            sys.executable,
```

Comment:
sys.executable may not point to python interpreter in certain circumstances. Community offered a solution using a sys.prefix which seems works reliably:

# https://bugzilla.mozilla.org/show_bug.cgi?id=1436777#c5
os.path.join(sys.prefix, 'bin', 'python')          # linux/osx
os.path.join(sys.prefix, 'Scripts', 'python.exe')  # Windows